### PR TITLE
Track updates of pre-commit hooks

### PR DESCRIPTION
**Pull request by ggravlingen**
I had a look to see if dependabot could keep the pre-commit hooks update. Apparently it wasn't in their roadmap, but I found this instead:
https://browniebroke.com/blog/gh-action-pre-commit-autoupdate/

### CODE REVIEW

File: .github/workflows/pre-commit-update.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        with:
+          python-version: 3.8
+      - name: Install pre-commit
+        run: pip install pre-commit
+      - name: Run pre-commit autoupdate
+        run: pre-commit autoupdate
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v2
+        with:
+          token: ${{ secrets.CPR_GITHUB_TOKEN }}
```
Comment (MartinHjelmare) [inline .github/workflows/pre-commit-update.yml:23]:
```suggestion
          token: ${{ secrets.GITHUB_TOKEN }}
```

File: .github/workflows/pre-commit-update.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+on:
+  schedule:
+    # Run on mondays
+    - cron: "0 0 * * 1"
+
+jobs:
+  auto-update:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```
Comment (MartinHjelmare) [inline .github/workflows/pre-commit-update.yml:11]: Please pin all action versions to the full semantic version.
```suggestion
      - uses: actions/checkout@v2.0.0
```

File: .github/workflows/pre-commit-update.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    # Run on mondays
+    - cron: "0 0 * * 1"
+
+jobs:
+  auto-update:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.0.0
+      - name: Set up Python
+        uses: actions/setup-python@v2
```
Comment (MartinHjelmare) [inline .github/workflows/pre-commit-update.yml:13]: Please update all version tags to the full semantic version.
```suggestion
        uses: actions/setup-python@v2.0.0
```

File: .github/workflows/pre-commit-update.yml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+jobs:
+  auto-update:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.0.0
+      - name: Set up Python
+        uses: actions/setup-python@v2.0.0
+        with:
+          python-version: 3.8
```
Comment (ggravlingen) [inline .github/workflows/pre-commit-update.yml:15]: Pin minor here too?

Comment (MartinHjelmare) [reply .github/workflows/pre-commit-update.yml:15]: No, we don't do that.

**Review by MartinHjelmare — APPROVED** [review]
Looks good!

**Pull request merged**